### ISSUE TITLE
`NotParsableFetchError`s lead to output even when handled

### ISSUE THREAD

**Issue by fmarier** [jvanasco/metadata_parser#32]
Because of an [explicit `log.error()`](https://github.com/jvanasco/metadata_parser/blob/223833b6c946d27253ba1b5e08c243baa642d7f8/metadata_parser/__init__.py#L1673), it's not possible to silence the `NotParsableFetchError` exception even when catching it in a `try..except` statement:
```
try:
    page = metadata_parser.MetadataParser(url=url, requests_timeout=5)
except metadata_parser.NotParsableFetchError as e:
    if e.code and e.code not in (403, 429, 500):
        print("Error parsing: %s [%s]" % (url, e.code))
```

**Comment by jvanasco** [comment]
This is by design. It was implemented on purpose for troubleshooting. You could turn off logging for the module, or raise the log level to critical - though I don't think anything logs to `critical` right now. Looking at this code, it should explicitly log the failed URL.

I am open to dropping this to `INFO` (which is what BeautifulSoup will create errors on) or `WARNING` if that works for your needs.

**Comment by fmarier** [comment]
Thanks for the clarification.

Dropping to `INFO` or `WARNING` would work. It would keep it out of tools like Sentry by default without having to adjust the module logging manually.

**Comment by jvanasco** [comment]
I've merged #34 for now, but am holding off on a release and may revert.

Why?

I spent some time today on clarifying the documentation and standardizing the use of logging.  After more reflection, I am not sure these types of errors encountered during the parsing and fetching should be logged at INFO instead of ERROR. 

Going by the standards of most of the other projects I contribute to or use, and even Python's own guidelines, I am not sure INFO is appropriate.

For example, https://docs.python.org/3/howto/logging.html states ERROR should be used "Due to a more serious problem, the software has not been able to perform some function" while INFO is "Confirmation that things are working as expected". 

I need to talk with some colleagues and users of this package about this before I proceed towards a release.
